roles/artifacts/tasks/edpm.yml

```diff
@@ -23,31 +23,32 @@
   when:
     - not cifmw_is_nested_virt | default(false) | bool
   block:
-    - name: Check for CI env directory
-      register: network_env_dir
-      ansible.builtin.stat:
-        path: /etc/ci/env
+    - name: Slurp zuul inventory file
+      ansible.builtin.slurp:
+        path: "{{ ansible_user_dir }}/ci-framework-data/artifacts/zuul_inventory.yml"
```

Comment:
FTR: This thing is not gonna work with BM nodes

Reply:
For BM Jobs, We use [`cifmw_is_nested_virt`](https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/artifacts/tasks/edpm.yml#L4). The above change is not applied to BM job. I will update the BM job template to run BM job on artifacts role changes.

Reply:
FTR: We will check if we can get rid off this and just use the sos-reports, embedded in must-gather.
I was referring to real BM jobs here.